core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1803,6 +1803,16 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  private[spark] val ARCHIVES = ConfigBuilder("spark.archives")
+    .version("3.1.0")
+    .doc("Comma separated list of archives to be extracted into the working directory of each " +
```

Comment:
nit: `Comma separated` -> `Comma-separated` for consistency?

Reply:
btw, we don't need to describe something here about the relationship (or difference?) between `park.files` and `spark.archives`?

Reply:
I think it's fine. They are meant to be separate configurations to end-users. These were not documented as well at least for `spark.files`, `spark.jars`, `spark.yarn.dist.files`, `spark.yarn.dist.archive`, `spark.yarn.dist.jars`, etc.